book/website/community-handbook/accessibility/accessibility-comms-platforms.md

```diff
@@ -0,0 +1,18 @@
+(ch-accessibility-comms-platforms)=
+# Access-centered support on Communication Platforms
+
+_The Turing Way_ uses a variety of platforms for communication (for example, social media platforms like X/Twitter, Linkedin, Mastodon) and collaboration (for example, Github, HackMD, Etherpad).
+This chapter describes the access and inclusion-related support we provide on our communication platforms, as well as support we hope to provide in the future.
+We encourage our community to practice kindness in communicating with others and to be aware of cultural, linguistic, disability and personal differences, including the potential digital divide (in terms of unequal access to, use or impact of Information and Communication Technologies (ICT)) that may affect the way people communicate online.
+On every platform our community uses to communicate or collaborate, we aim to enable access in a broader sense as defined by our Accessibility Policy.
+
+## Access and inclusion-related support we provide on communication platforms
+
+We aim to provide the following support and normalise the following practices within our community:
+
+- **Adding alt text to images**: We use alt text or "alternative text" to describe images, diagrams, or other visuals within _The Turing Way_. To learn more about how to write and use alt text for your work, please use this {ref}`guidance<ch-accessibility-alttext>`.
+- **Limit embedded links and provide alternatives for accessing information**: On many platforms, embedded links may not be accessible by screen readers, or may increase cognitive overload. If you are inserting a link into a message, limit the number of embedded links within a message. On Slack, avoid hyperlinking links within replies to messages when possible, as links posted within threads are not readable for screen readers.
+- **Limit use of emojis that disrupt narrative flow**: Do not replace words with emojis or use too many emojis to communicate important information. Screen readers usually read one emoji at a time using the alternative text and while sometimes the purpose may be clear (```:thumbsup:``` represents an icon of a thumbs up sign), at other times it may not be (```:partyblobcat``` represents a multicolored cat that is rotating in a circle, and is generally used in the context of a celebration or celebratory news). More broadly, strings of emojis can produce a clunky, repetitive user experience for people trying to understand the core message.  Remember that not all people are able to access the emoji reactions option within Slack, and normalise using written affirmations as well.
```

Comment:
```suggestion
- **Limit use of emojis that disrupt narrative flow**: Do not replace words with emojis or use too many emojis to communicate important information. 
Screen readers usually read one emoji at a time using the alternative text and while sometimes the purpose may be clear (```:thumbsup:``` represents an icon of a thumbs up sign), at other times it may not be (```:partyblobcat``` represents a multicolored cat that is rotating in a circle, and is generally used in the context of a celebration or celebratory news). 
More broadly, strings of emojis can produce a clunky, repetitive user experience for people trying to understand the core message.
Remember that not all people are able to access the emoji reactions option within Slack, and normalise using written affirmations as well.
```